src/presenter/trip-point-board-presenter.js

```diff
@@ -86,6 +89,14 @@ export default class TripPointBoardPresenter{
     }
   }
 
+  #renderNewPoint() {
+    const blankPoint = this.#tripPointsModel.blankPoint;
+    const pointPresenter = new TripPointPresenter({offers:this.#offers, destinations:this.#destinations, tripEventsListContainer:this.#tripEventsListContainer, handelPointChange:this.#handelPointChange, handelTypeChange:this.#handleTypeChange});
```

Comment:
```suggestion
    const pointPresenter = new TripPointPresenter({
      offers: this.#offers,
      destinations: this.#destinations,
      tripEventsListContainer: this.#tripEventsListContainer,
      handelPointChange: this.#handelPointChange,
      handelTypeChange: this.#handleTypeChange,
    });
```
Немного форматирования не помешает в таких кейсах.